src/Task/Extra.elm

```diff
@@ -8,13 +8,16 @@ module Task.Extra exposing
     , map12
     , map13
     , map14
+    , andMap
     )
 
 {-|
 
 
 # Task.Extra
 
+Functions provided by this module execute Tasks in sequence one after another.
```

Comment:
I think it's fair to mention this in documentation because it would be theoretically possible to implement concurrent handling of tasks (in [`Aff monad`](https://pursuit.purescript.org/packages/purescript-aff/5.1.2) style)  if elm's Task effect manager would allow for that (which it doesn't)